src/notification-toast/notification-toast.styles.tsx

```diff
@@ -0,0 +1,181 @@
+                iconColor = Color.Validation.Green.Icon(props);
+                break;
+            case "error":
+                backgroundColor = Color.Validation.Red.Background(props);
+                borderColor = Color.Validation.Red.Border(props);
+                textColor = Color.Validation.Red.Text(props);
+                iconColor = Color.Validation.Red.Icon(props);
+                break;
+            case "warning":
+                backgroundColor = Color.Validation.Orange.Background(props);
+                borderColor = Color.Validation.Orange.Border(props);
+                textColor = Color.Validation.Orange.Text(props);
+                iconColor = Color.Validation.Orange.Icon(props);
+                break;
+            case "info":
+                backgroundColor = Color.Validation.Blue.Background(props);
+                borderColor = Color.Validation.Blue.Border(props);
+                textColor = Color.Validation.Blue.Text(props);
+                break;
+            case "successDark":
+                backgroundColor = Color.Validation.DarkMode.Background(props);
+                borderColor = Color.Validation.DarkMode.Border(props);
+                textColor = Color.Validation.DarkMode.Text(props);
+                iconColor = Color.Validation.Green.Icon(props);
+                break;
+            case "errorDark":
+                backgroundColor = Color.Validation.DarkMode.Background(props);
+                borderColor = Color.Validation.DarkMode.Border(props);
+                textColor = Color.Validation.DarkMode.Text(props);
+                iconColor = Color.Validation.Red.Border(props);
+                break;
+            case "warningDark":
+                backgroundColor = Color.Validation.DarkMode.Background(props);
+                borderColor = Color.Validation.DarkMode.Border(props);
+                textColor = Color.Validation.DarkMode.Text(props);
+                iconColor = Color.Validation.Orange.Icon(props);
+                break;
+            case "infoDark":
+                backgroundColor = Color.Validation.DarkMode.Background(props);
+                borderColor = Color.Validation.DarkMode.Border(props);
+                textColor = Color.Validation.DarkMode.Text(props);
+                iconColor = Color.Validation.Blue.Icon(props);
+                break;
+            default:
+                backgroundColor = Color.Validation.Green.Background(props);
+                borderColor = Color.Validation.Green.Border(props);
+                textColor = Color.Validation.Green.Text(props);
+                iconColor = Color.Validation.Green.Icon(props);
+                break;
+        }
+
+        return `
+			background: ${backgroundColor};
+			border: 1pt solid ${borderColor};
+            color: ${textColor};    
+
+          svg {
+            color: ${iconColor};
+            font-size: 24px;
+            margin-right: 4px;  
+        }
+
+        p {
```

Comment:
I see the text style being set in multiple places. You might want to mandate a consistent way of specifying.

Reply:
what i mean by mandate a consistent way of specifying is, likely the text content will come via the `children` prop. If it is the case, then you should style the nearest parent component instead.